core/src/language/python.rs

```diff
@@ -754,4 +742,310 @@ mod test {
         python.write_field(mock_writer, &rust_field, &[]).unwrap();
         assert_eq!(String::from_utf8_lossy(mock_writer), "    field: str\n");
     }
+
+    #[test]
+    fn simple_test_tagged_enum() {
+        use serde::{Deserialize, Serialize};
+        let mut python = Python::default();
+        #[derive(Serialize, Deserialize)]
+        #[serde(tag = "test_tag_key", content = "test_content_key")]
+        enum Test {
+            Unit,
+            Tuple(i32),
+            Struct {
+                #[serde(default)]
+                field: String,
+            },
+        }
+
+        let shared_enum = RustEnumShared {
+            id: Id {
```

Comment:
Is there anyway we can use the `parse_enum` function inside the `parser.rs`? I am scared that directly instantiating the `RustEnumShared` is prone to human errors.